types/google.visualization/index.d.ts

```diff
@@ -67,10 +69,26 @@ declare namespace google {
             setOptions(options: Object): void;
             setView(view_spec: string): void;
         }
-
         //#endregion
+        
+        //#region data
+        export class data {
+            // https://developers.google.com/chart/interactive/docs/reference#group
+            static group(data: DataTable, keys: any[], columns: any[]): DataTable;
+            static group(data: DataView, keys: any[], columns: any[]): DataTable;
+            
+            static sum(values: number[]): number;
+            static avg(values: number[]): number;
+            static min(values: number[]): number;
+            static max(values: number[]): number;
+            static count(values: number[]): number;
```

Comment:
`values` here is type of `any`

Reply:
changed in the same commit as the other aggregates